_episodes/13-scheduler.md

```diff
@@ -63,7 +63,7 @@ whichever you prefer. Unsure? `nano` is a pretty good, basic choice.
 {: .bash}
 
 ```
-#!/bin/bash
+#!/bin/bash {{ site.sched.flag.bash }}
```

Comment:
Variable when any script is shown.

Reply:
This script runs on the landing pad (head node), and executes the system utility `hostname`. As discussed in #126, it seems extraneous here.